Sources/AblyChat/MessageReaction.swift

```diff
@@ -184,6 +190,23 @@ public struct MessageReactionSummary: Sendable, Equatable {
          * Map of clients who left given reaction type number of times.
          */
         public var clientIds: [String: UInt]
+
+        /**
+         * The sum of the counts from all unidentified clients who have published an annotation with this
+         * name, and so who are not included in the clientIds list
+         */
+        public var totalUnidentified: UInt // TM7d1d
```

Comment:
I missed the fact that we had at some point started using `UInt` in the public API. Swift discourages the usage of `UInt` in general (whereas it was more common in Objective-C). @maratal was this a change that you introduced?

Reply:
See https://docs.swift.org/swift-book/documentation/the-swift-programming-language/thebasics/#UInt

> Use UInt only when you specifically need an unsigned integer type with the same size as the platform’s native word size. If this isn’t the case, Int is preferred, even when the values to be stored are known to be nonnegative. A consistent use of Int for integer values aids code interoperability, avoids the need to convert between different number types, and matches integer type inference, as described in [Type Safety and Type Inference](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/thebasics#Type-Safety-and-Type-Inference).